keps/sig-node/3960-pod-lifecycle-sleep-action/kep.yaml

```diff
@@ -4,7 +4,7 @@ authors:
   - "@AxeZhan"
 owning-sig: sig-node
 participating-sigs: []
-status: implementable
+status: implemented
```

Comment:
Revert back to "implementable" - we switch to implemented only after the the release graduating to GA is already out.